common/model-views.cpp

```diff
@@ -2556,18 +2587,26 @@ namespace rs2
                     if (dev->post_processing_enabled)
```

Comment:
It is somewhat strange that the processing loop is not double-checking that the sensor is indeed stereo or depth device

Reply:
An initial qualification is in the beginning of the function:
> if (f.get_profile().stream_type() == RS2_STREAM_DEPTH)
The additional sanity is performed independently by each filter